### ISSUE TITLE
Invert y axis in lineChart

### ISSUE THREAD

**Issue by evanb2** [nvd3-community/nvd3#99]
So I finally was able to set the yAxis tickValues but now when I use yRange() to inverse it the ticks disappear, is there a better way of doing this? Or is there another setting I'm forgetting to add??

Here's the relevant code:
```javascript
var xLabels = ["", 250, 500, 750, 1000, 1500, 2000, 3000, 4000, 6000, 8000, 10000];
var yLabels = [-10, 0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120];
var chart = nv.models.lineChart()
        .x(function (d) {
            return d.Hz;
        })
        .y(function (d) {
            return d.dB;
        })
        .showLegend(false)
        .showYAxis(true)
        .showXAxis(true);
chart.forceY(yLabels);
chart.forceX(xLabels);
chart.yRange([0, 450]);

chart.xAxis
        .axisLabel('Frequency (Hz)')
        .tickValues(xLabels)
        .orient('top');

chart.yAxis
        .axisLabel('dB')
        .tickValues(yLabels);
```

**Comment by evanb2** [comment]
![screen shot 2016-02-23 at 4 44 49 pm](https://cloud.githubusercontent.com/assets/11276439/13271894/f95a93f0-da4c-11e5-9845-1fd5855f62a1.png)

This is what I'm trying to build.

**Comment by evanb2** [comment]
So I did finally figure out that the yAxis ticks were there the whole time, but for some reason NVD3 was setting their opacity = 0...so I just couldn't see them. I set the `<text>` opacity = 1 and they appeared.

I still can't figure out how to set the xAxis like the picture above...I can set the tickValues but the spacing is all messed up because it's trying to make a linear scale when I really want an ordinal scale.

If anyone is actually reading this some help would be appreciated.
